File: dayFive/javaScript/addDigits.js
function sumTheDigitsToOneDigit(number){
    if(number < 0){
        number = 0 - number;
    }

    let singleDigit = 0;
    while(number > 9){
        singleDigit = Math.floor(number / 10) + (number % 10);
        number = singleDigit;
    }
    return singleDigit;
}

let value = 38;
console.log(sumTheDigitsToOneDigit(value));

File: dayFive/python/add_digits.py
def sumTheDigitsToOneDigit(number):
    if(number < 0):
        number = 0 - number
    
    singleDigit = 0
    while(number >= 10):
        singleDigit = (number // 10) + (number % 10)
        number = singleDigit
    
    return singleDigit


value = 38
print(sumTheDigitsToOneDigit(value))

